### DISCUSSION TITLE
Collection Values not working

### DISCUSSION THREAD

**Discussion by stayupthetree** [joalla/discogs_client#139] [Support] [answered]
I found this [discussion](https://github.com/joalla/discogs_client/discussions/54) from a couple of years ago and attempted to implement it, but get the following error. Is collection value no longer a thing?

Traceback (most recent call last):
  File "/home/tree/discogs/Discogs.py", line 161, in <module>
    main()
  File "/home/tree/discogs/Discogs.py", line 151, in main
    see_collection_value()
  File "/home/tree/discogs/Discogs.py", line 123, in see_collection_value
    collection_value = me.collection_value
AttributeError: 'User' object has no attribute 'collection_value'. Did you mean: 'collection_folders'?

def see_collection_value():
    me = d.identity()
    collection_value = me.collection_value
    
    # Displaying median, maximum, and minimum values of the collection
    print(f"Median Collection Value: {collection_value.median}")
    print(f"Maximum Collection Value: {collection_value.maximum}")
    print(f"Minimum Collection Value: {collection_value.minimum}")

**Comment by AnssiAhola** [comment]
Hey @stayupthetree, which version of the client are you using? the latest (2.7)?
I'm not on my main pc atm, so can't test the code, but will report back if there's indeed a bug with the latest version.

**Reply by JOJ0** [reply]
Hi @stayupthetree did that solve it? Was it the version? A common issue could be u using discogs-client instead of python3-discogs-client package. HTH

**Comment by JOJ0** [comment, marked as answer]
collection value works, just tested
